components/studio/build-docker-image.ps1

```diff
@@ -27,10 +27,15 @@ try {
         $InstallHarts = @(
             "core/hab-studio",
             "core/hab-sup",
-            "core/windows-service"
```

Comment:
Do `hab-studio` and `hab-sup` not also need to be pulled from `stable`, too? If not, what's the difference between how these packages are used?

Reply:
No we want those versions to align with whatever channel we are building (unstable or rc). `windows-service` is different because it is maintained in a separate repo and changes very infrequently. Until this last release it was not used by any of our core code but was instead a sort of "add-on" that windows users could use to harness the launcher/supervisor as a windows service.

One option would be to bring it in to this repo but I don't see that as particularly high value yet.

Reply:
Ah, gotcha. That makes sense. Could you basically paste that response into the code as a comment so we keep that rationale around? I could see someone accidentally changing that in some future refactoring, absent that information.